clients/src/test/java/org/apache/kafka/clients/consumer/internals/AbstractCoordinatorTest.java

```diff
@@ -498,33 +498,6 @@ public void testHandleLeaveGroupResponseWithException() {
         assertTrue(leaveGroupFuture.exception() instanceof UnknownMemberIdException);
     }
 
-    @Test
-    public void testHandleSingleLeaveGroupRequest() {
```

Comment:
Found this test an exact duplicate of `testHandleNormalLeaveGroupResponse` so removing.